### PR TITLE
Yandex Metrika feature

### PR DESCRIPTION
**Pull request by ruslankonev**

### CODE REVIEW

File: modules/yandex-metrika/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const fs = require('fs-extra')
+const path = require('path')
+
+module.exports = function nuxtAnalytics(options) {
```
Comment (pi0) [inline modules/yandex-metrika/index.js]: need to change this

File: modules/yandex-metrika/index.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  // Don't include on dev mode
+  if (this.options.dev && process.env.NODE_ENV !== 'production') {
+    return
+  }
+
+  // Add google analytics script to head
+  this.options.head.script.push({
+    src: options.analyticsURL || 'https://mc.yandex.ru/metrika/watch.js',
```
Comment (pi0) [inline modules/yandex-metrika/index.js]: also this :)

File: modules/yandex-metrika/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Yandex Metrika
+Add Yandex Metrika to your nuxt.js application.
```
Comment (pi0) [inline modules/yandex-metrika/README.md]: Would you please provide a link to matrika docs too?

**Comment by ruslankonev** [comment]
Updated

**Comment by pi0** [comment]
Thanks @ruslankonev  ;)

**Pull request merged**